wdl_runner/cromwell_driver.py

```diff
@@ -44,7 +44,8 @@ def start(self):
         jvm_args += ["-jar", self.cromwell_jar]
         self.cromwell_proc = subprocess.Popen(["java"] + jvm_args + ["server"])
 
-        logging.info("Started Cromwell")
+        logging.info("Started Cromwell using command: %s", " ".join(["java"] + jvm_args
```

Comment:
thanks for the comment @ovesh 
1. would you mind to elaborate a bit more on "Cache the list passed to Popen? (so that if it changes you don't need to also change this line)"? 🤔

Reply:
Something like this
```
        cmd = ["java"] + jvm_args + ["server"]
        self.cromwell_proc = subprocess.Popen(cmd)
        ....
        logging.info("Started Cromwell using command: %s", " ".join(cmd)
```

Reply:
fixed in [c3b23d1](https://github.com/deepgenomics/wdl-runner/pull/10/commits/c3b23d1323907deecc04bf7dd3f29b1be9a26169)